config/crd/bases/anywhere.eks.amazonaws.com_bundles.yaml

```diff
@@ -42,7 +42,7 @@ spec:
                 type: string
               number:
                 description: Monotonically increasing release number
-                type: integer
+                type: string
```

Comment:
Don't you need to bump the apiVersion when you do stuff like this?

Reply:
I believe we would do that for bigger (possibly breeaking) changes and in case we revamped the API considerably, introduction of newer suite of API types, etc.